#175 draft_id を Exhibition に含めた

su8ru opened the pull request:
close #175 

ちなみに BlogArticle には revision_id が含まれてます そのへんも踏まえての追加判断

```ts
export interface BlogArticle {
  // ...
  revision_id: number;
  // ...
}

```

Takeno-hito commented:
Revisionのフィルターにpublished, deletedフィルターが存在するけどそれでは足りない？特定のrevision探しには回りくどくなるかもしれないけどpublished & !deleted がいける

su8ru commented:
> Revisionのフィルターにpublished, deletedフィルターが存在するけどそれでは足りない？特定のrevision探しには回りくどくなるかもしれないけどpublished & !deleted がいける
@Takeno-hito 

回りくどい、HTTP リクエストが増える、実装が非直感的になる
draft_id を実装しない積極的理由ではないだろ
